hw6.py

```diff
@@ -0,0 +1,98 @@
+class Exchanger(object):
+    _base = None
+
+    @staticmethod
+    def exchange(money, to_currency="USD"):
+        ratio = Exchanger._base.get((money.currency, to_currency))
+        if ratio is None:
+            # print("{} to {} ratio hasn't found! Used 1.0 as the ratio.")
+            ratio = 1.0
+        return Money(money.amount * ratio, to_currency)
+
+
+class ExchangerOnline(Exchanger):
+    @staticmethod
+    def exchange(money, to_currency="USD"):
+        if Exchanger._base is None:  # if dictionary is empty
```

Comment:
Actually, `is None` verifies that `base` is not initialized.
`base = {}`  <--- empty dict